index.yaml

```diff
@@ -3,60 +3,60 @@ entries:
   hypershift-agent-template:
   - apiVersion: v2
     appVersion: 0.0.1
-    created: "2023-06-15T12:57:53.103446028+02:00"
+    created: "2023-07-17T11:40:30.088882972+03:00"
     description: Hypershift cluster template with agent workers
-    digest: 155c71eb3def8c73228b0e748206705e2d3c21f08043b69be998885bd1dd0a4a
+    digest: 6c102d58ca9a76c17681eb260e71be3e699801f292501d225261356cb3d6f8de
     name: hypershift-agent-template
     type: application
     urls:
-    - https://stolostron.github.io/cluster-templates-manifests/hypershift-agent-template-0.0.1.tgz
+    - https://batzionb.github.io/cluster-templates-manifests/hypershift-agent-template-0.0.1.tgz
     version: 0.0.1
   hypershift-kubevirt-template:
   - annotations:
       cluster-template: "true"
     apiVersion: v2
-    created: "2023-06-15T12:57:53.103862936+02:00"
+    created: "2023-07-17T11:40:30.08928852+03:00"
     description: A Helm chart which installs Hypershift cluster using KubeVirt platform
     digest: 2b5d44df585930cc6b4c4129937b0434e8ebc957d08f6d084ae8d501f1dcabcb
     name: hypershift-kubevirt-template
     type: application
     urls:
-    - https://stolostron.github.io/cluster-templates-manifests/hypershift-kubevirt-template-0.0.3.tgz
+    - https://batzionb.github.io/cluster-templates-manifests/hypershift-kubevirt-template-0.0.3.tgz
```

Comment:
can you please move back to `stolostron`?

Reply:
I removed the changes in the index.yaml all together
seems like it doesn't need reindexing since there's no version added